### PR TITLE
Floating label in liveform and livefilter

### PR DESCRIPTION
**Pull request by aparnasj**

### CODE REVIEW

**Review by swethakundaram — APPROVED** [review]

**Comment by prasanth-reddy-wavemaker-com** [comment]
Lets wait till we release 10.6.2 before we merge this PR

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ Live Form **layouts** include:
 1. Column Form with left-aligned Captions
 2. Column Form with right-aligned Captions
 3. Column Form with top-aligned Captions
+4. Column Form with floating Captions
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/live-form/liveform-layouts.md:14]: Lets add in brackets (Angular material style)

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md [hunk 2]
```diff
@@ -43,5 +45,9 @@ For form fields, the **alignment of the caption** can be configured to place as
 ### 3. Column Form with top-aligned Captions 
     
 <iframe width="100%" height="450" style="background-color: snow;" allowtransparency="true" src="https://apps.wavemakeronline.com/documentation_snippets/#/LiveFormwithThreeCol">3-column Form</iframe>
-    
+
+### 4. Column Form with floating Captions
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/live-form/liveform-layouts.md:49]: Lets add in brackets (Angular material style)

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md (hunk 1 above)
Comment (aparnasj) [reply learn/app-development/widgets/datalive/live-form/liveform-layouts.md]: Added

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md (hunk 2 above)
Comment (aparnasj) [reply learn/app-development/widgets/datalive/live-form/liveform-layouts.md]: Added

File: learn/app-development/widgets/datalive/form/form-layouts.md [hunk 3]
```diff
@@ -5,9 +5,10 @@ sidebar_label: "Layouts"
 ---
 ---
 
-Form **layout** can be _1-column_, _2-column_ or _3-column_. Apart from this, the **alignment of the caption** for the form fields can be configured to be placed at the _top_, _left_ or _right_ of the form fields.
+Form **layout** can be _1-column_, _2-column_ or _3-column_. Apart from this, the **alignment of the caption** for the form fields can be configured to be placed at the _top_, _left_, _right_ or _floating_ of the form fields.
 
 1. 1-Column Form with left-aligned Captions
 2. 2-Column Form with right-aligned Captions
 3. 3-Column Form with top-aligned Captions
+4. 3-Column Form with floating Captions (Angular material style)
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/form/form-layouts.md:13]: Link Angular material style to this https://material.angular.io/components/form-field/examples

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md [hunk 4]
```diff
@@ -43,5 +45,9 @@ For form fields, the **alignment of the caption** can be configured to place as
 ### 3. Column Form with top-aligned Captions 
     
 <iframe width="100%" height="450" style="background-color: snow;" allowtransparency="true" src="https://apps.wavemakeronline.com/documentation_snippets/#/LiveFormwithThreeCol">3-column Form</iframe>
-    
+
+### 4. Column Form with floating Captions (Angular material style)
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/app-development/widgets/datalive/live-form/liveform-layouts.md:49]: Link Angular material style to this https://material.angular.io/components/form-field/examples

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/app-development/widgets/datalive/live-form/liveform-layouts.md (hunk 4 above)
Comment (aparnasj) [reply learn/app-development/widgets/datalive/live-form/liveform-layouts.md:49]: done

File: learn/app-development/widgets/datalive/form/form-layouts.md (hunk 3 above)
Comment (aparnasj) [reply learn/app-development/widgets/datalive/form/form-layouts.md]: done

**Pull request merged**